RKNotificationHub/RKNotificationHub.h

```diff
@@ -41,20 +46,19 @@
 - (id)initWithBarButtonItem:(UIBarButtonItem *)barButtonItem;
 
 //%%% adjustment methods
-- (void)setView:(UIView *)view andCount:(int)startCount;
+- (void)setView:(UIView *)view andCount:(NSUInteger)startCount;
 - (void)setCircleAtFrame:(CGRect)frame;
 - (void)setCircleColor:(UIColor*)circleColor labelColor:(UIColor*)labelColor;
 - (void)moveCircleByX:(CGFloat)x Y:(CGFloat)y;
 - (void)scaleCircleSizeBy:(CGFloat)scale;
-- (void)setCountLabelFont:(UIFont *)font;
+@property (nonatomic, strong) UIFont *countLabelFont;
```

Comment:
At the time, I'd thought I'd want a way to get the value of `countLabelFont` and this `@property` line is equivalent to declaring the setter that was already there plus the getter I wanted.